modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx

```diff
@@ -129,41 +129,52 @@ const CustomView: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 
 	const handleSaveObjectView = async () => {
 		const newObjectView = removeLabelFromObjectView(objectView);
+		const {objectViewColumns} = newObjectView;
 
-		const response = await Liferay.Util.fetch(
-			`/o/object-admin/v1.0/object-views/${objectViewId}`,
-			{
-				body: JSON.stringify(newObjectView),
-				headers: HEADERS,
-				method: 'PUT',
-			}
-		);
-
-		if (response.status === 401) {
-			window.location.reload();
-		}
-		else if (response.ok) {
+		if (objectView.defaultObjectView && objectViewColumns.length === 0) {
```

Comment:
Hi @danvb112, could you invert the condition? This way the request is made first and `else` should fire alert

Reply:
I agree with you @interaminense , I hadn't thought of that when I made the solution, Thanks! :)